zxlive/mainwindow.py

```diff
@@ -182,6 +186,8 @@ def __init__(self) -> None:
         self.effects = {e: load_sfx(e) for e in SFXEnum}
 
         QShortcut(QKeySequence("Ctrl+B"), self).activated.connect(self._toggle_sfx)
+        
+        get_local_api_key() # load the local user's API key
```

Comment:
I think this will currently give a fatal error if the key file does not exist. So then the file should be added to the manifest. Better yet would be to load the key from the ZXLive Qt Settings.